modules/Bio/EnsEMBL/Analysis/Hive/Config/StarScallopRnaseq.pm

```diff
@@ -0,0 +1,827 @@
+    'databases_port'            => '',                                                                                # port for general output db host
+    'dna_db_port'               => '',                                                                                # port for dna db host
+    'registry_host'             => '',                                                                                # host for registry db
+    'registry_port'             => '',                                                                                # port for registry db
+    'registry_db'               => '',                                                                                # name for registry db
+    'rnaseq_summary_file'       => '' || catfile( $self->o('rnaseq_dir'), $self->o('species_name') . '.csv' ),        # Set this if you have a pre-existing cvs file with the expected columns
+    'rnaseq_summary_file_genus' => '' || catfile( $self->o('rnaseq_dir'), $self->o('species_name') . '_gen.csv' ),    # Set this if you have a pre-existing genus level cvs file with the expected columns
+    'release_number' => '' || $self->o('ensembl_release'),
+    'species_name'        => '',                                                                                      # e.g. mus_musculus
+    'production_name'     => '',                                                                                      # usually the same as species name but currently needs to be a unique entry for the production db, used in all core-like db names
+    'taxon_id'            => '',                                                                                      # should be in the assembly report file
+    'uniprot_set'         => '',                                                                                      # e.g. mammals_basic, check UniProtCladeDownloadStatic.pm module in hive config dir for suitable set,
+    'output_path'         => '',                                                                                      # Lustre output dir. This will be the primary dir to house the assembly info and various things from analyses
+    'assembly_name'       => '',                                                                                      # Name (as it appears in the assembly report file)
+    'registry_file'       => '' || catfile( $self->o('output_path'), "Databases.pm" ),                                # Path to databse registry for LastaZ and Production sync
+    'use_genome_flatfile' => '1',                                                                                     # This will read sequence where possible from a dumped flatfile instead of the core db
+    'skip_rnaseq'         => '0',                                                                                     # Will skip rnaseq analyses if 1
```

Comment:
This is not needed here but in the main pipeline. It can be removed

Reply:
Removed both vars.

Reply:
Only skip_rnaseq is to be removed

Reply:
I couldn't find any reference to use_genome_flatfile or registry_file in the analyses. Should we add them back anyway?

Reply:
Yeah it can be removed if they are not referenced